ord_schema/validations.py

```diff
@@ -1067,9 +1067,9 @@ def validate_data(message):
         validate_reaction_workup,
     reaction_pb2.ReactionOutcome:
         validate_reaction_outcome,
-    reaction_pb2.ReactionProduct:
+    reaction_pb2.ProductCompound:
         validate_reaction_product,
```

Comment:
If I'm allowed to be picky, can you rename `validate_reaction_product` to `validate_product_compound`?

Reply:
Absolutely! Fixed.